app/build.gradle

```diff
@@ -0,0 +1,73 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 26
+    flavorDimensions "version"
+
+    signingConfigs {
+        releaseconfig {
+            keyAlias 'recipe'
+            keyPassword 'application'
+            storeFile file('/home/shubhank/AndroidStudioProjects/RecipeApplication1/app/Production/SignedAPk/production-release.jks')
```

Comment:
Did you add this path?

Reply:
i am not added.When generate singed apk it is set path